module/src/main/java/org/mule/extension/spring/internal/SpringModuleExtensionModelGenerator.java

```diff
@@ -71,6 +73,23 @@ public void accept(ExtensionDeclarer extensionDeclarer, ExtensionLoadingContext
     parameterGroupDeclarer.withRequiredParameter("files").withExpressionSupport(NOT_SUPPORTED)
         .withRole(BEHAVIOUR).ofType(typeLoader.load(String.class));
 
+    extensionDeclarer.withExternalLibrary(ExternalLibraryModel.builder()
```

Comment:
Review to only ask for the mandatory dependencies

Reply:
done